integration_tests/conftest.py

```diff
@@ -776,17 +776,19 @@ def pred_semantic_segs(
 
 
 @pytest.fixture
-def pred_clfs(model_name: str, img5: Datum, img6: Datum) -> list[Prediction]:
+def pred_clfs(
+    model_name: str, img5: Datum, img6: Datum, img8: Datum
+) -> list[Prediction]:
     return [
         Prediction(
             datum=img5,
             annotations=[
                 Annotation(
                     task_type=TaskType.CLASSIFICATION,
                     labels=[
-                        Label(key="k12", value="v12", score=0.47),
```

Comment:
the existing tests violated our new validation check, so I had to rewrite these test values and downstream assertions